### ISSUE TITLE
deployment sucks

### ISSUE THREAD

**Issue by quintinappi** [stackblitz/bolt.new#416]
### Describe the bug

no deployment that is claimed is ever kept, every new deployment deploys into another universe and doesnt go to my netlify account and update that deployment since ive renamed my app on netlify

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-x4jhci

### Steps to reproduce

1. click deploy
2. wait for deploy ( most of the times it producers an error instead of working )
3. if it does work it does a deploy into a new universe

### Expected behavior

I expect when claiming the deployment address, assigninig it to my netlify account, changing the site name, that any further deployments will fall under that account

### Platform

```
Browser name = Chrome
Full version = 129.0.0.0
Major version = 129
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 57412645,
  "usedJSHeapSize": 51983269,
  "jsHeapSizeLimit": 4294705152
}
Username = quintinappi
Chat ID = 9ce7b35168dc
```

**Comment by kc0tlh** [comment]
Thanks for the report and patience during this beta period! Tracking all deployment related bug fixes in issue #499

**Comment by valdeno974** [comment]
Same for me, I never receive the "claim url".
The project seems to disappear between Bolt.new and Netlify.
